src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -3,12 +3,6 @@ var FEEDBACK_RESPONSE_TEXT = 'responsetext';
 var FEEDBACK_MISSING_RECIPIENT = 'You did not specify a recipient for your response in question(s)';
 var WARNING_STATUS_MESSAGE = '.alert-warning.statusMessage';
 
-// selectors for warning modal prompt and the various fields within
-var WARNING_MODAL = '#warning-modal';
-var WARNING_MODAL_TITLE = '#warning-modal-title';
-var WARNING_MODAL_MESSAGE = '#warning-modal-message';
-var WARNING_MODAL_OK = '#warning-modal-ok';
-
 // text displayed to user
 var SESSION_NOT_OPEN = 'Feedback Session Not Open';
 var OKAY = 'Okay';
```

Comment:
Something like this. In other words you're doing `'Okay' || 'OK'` in the JS :joy: